monkestation/_maps/random_rooms.dm

```diff
@@ -0,0 +1,76 @@
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
+	mappath = "monkestation/_maps/RandomRooms/10x10/skwh_rdm158_abandonedviro.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 3
+
+/datum/map_template/random_room/skwh_rdm157
+	name = "Slime Storage"
+	room_id = "skwh_rdm157_slimestorage"
+	mappath = "monkestation/_maps/RandomRooms/10x5/skwh_rdm157_slimestorage.dmm"
+	centerspawner = FALSE
+	template_height = 5
+	template_width = 10
+	weight = 3
+
+/datum/map_template/random_room/skwh_rdm160
+	name = "Green Viney Riches"
+	room_id = "skwh_rdm160_greenvineyriches"
+	mappath = "monkestation/_maps/RandomRooms/10x10/skwh_rdm160_greenvineyriches.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 6
```

Comment:
this is WAY too many kudzu seeds, especially for a _weight six_ room.

Reply:
weird for some reason i remember giving this room a weight 2 or 1 but put 6 by accident, by this point i think i might just remove the room due to the amount of kudzu seeds, i can understand having the kudzu seeds of the slime storage removed, i mean i mainly just added those seeds to that room as just some random garbage players can get on the floor, but yeah